packages/perseus/src/widgets/__stories__/test-keypad-context-wrapper.jsx

```diff
@@ -4,60 +4,11 @@ import {View} from "@khanacademy/wonder-blocks-core";
 import {StyleSheet} from "aphrodite";
 import * as React from "react";
 
-import {ItemRenderer} from "../../index.js";
```

Comment:
Originally I split this logic out so both `on-screen-keypad.stories.jsx` and `expression.stories.jsx` could use the same keypad provider. However (IMO) `on-screen-keypad.stories.jsx` was really just a very specific expression story, so I ended up just strengthening up `expression.stories.jsx` and getting rid of `on-screen-keypad.stories.jsx`.

Keeping this split out logic separate though, because (I think) we want the keypad in lots of components. Also I didn't want to clutter `expression.stories.jsx`.